webpack.config.js

```diff
@@ -205,27 +206,12 @@ function getWebpackConfig( {
 			// https://github.com/localForage/localForage/issues/577
 			noParse: /[\/\\]node_modules[\/\\]localforage[\/\\]dist[\/\\]localforage\.js$/,
 			rules: [
-				{
-					test: /\.jsx?$/,
-					exclude: /node_modules\//,
-					use: [
-						{
-							loader: 'thread-loader',
-							options: {
-								workers: workerCount,
-							},
-						},
-						{
-							loader: 'babel-loader',
-							options: {
-								configFile: path.resolve( __dirname, 'babel.config.js' ),
-								babelrc: false,
-								cacheDirectory: path.join( __dirname, 'build', '.babel-client-cache' ),
-								cacheIdentifier,
-							},
-						},
-					],
-				},
+				JsxConfig.loader( {
+					workerCount,
+					configFile: path.resolve( __dirname, 'babel.config.js' ),
```

Comment:
Let's be consistent in the `path` methods we use.

```suggestion
					configFile: path.join( __dirname, 'babel.config.js' ),
```